packages/block-template-react/.gitignore.dist

```diff
@@ -1,2 +1,3 @@
 dist
 node_modules
+.bprc
```

Comment:
`EOL` worth fixing given that it's a template

Reply:
You can avoid this in the future by adding `"files.insertFinalNewline": true,` to VSCode config

Reply:
https://github.com/blockprotocol/blockprotocol/pull/575/files/fdc585c6d6e709fbce49ca0060eb44baa4972bad..7666a5ab623fb4777c2ffe3a26276dbbc157eeaa

Reply:
https://github.com/blockprotocol/blockprotocol/pull/575/commits/2b6e49cd069970b0ae07b039aa9c09724750e0d0 now?

Reply:
can we do this automatically as part of formatting?